apps/reactotron-app/src/renderer/pages/reactNative/Overlay.tsx

```diff
@@ -123,8 +125,18 @@ function Overlay() {
     event.preventDefault()
     event.stopPropagation()
     if (event.dataTransfer.files.length !== 1) {
+      sendAnalyticsEvent({
+        category: "error",
+        action: "OverlayDropImage",
+        label: "Too many files",
+      })
       return
     }
+    sendAnalyticsEvent({
+      category: "overlay",
+      action: "OverlayDropImage",
+      label: "Success",
+    })
```

Comment:
I noticed that each `sendAnalyticsEvent` has string values for category, action, and label. It may be worth using an enum for these constants. Then it's type-safe and easier to see what is available per key! Thoughts?

Reply:
Thanks for the suggestion @jonmajorc! Yeah, i need to clean up the types for sending events so we know exactly what we're going to send to the analytics server.